README.md

```diff
@@ -15,21 +15,27 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
 ## Building locally
 
 - Install [Node.js](https://nodejs.org) version 16
-    - If you are using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) (recommended) running `nvm use` will automatically choose the right node version for you.
```

Comment:
Were these changes intentional? If Prettier necessitated these changes, then I'm wondering why we haven't had an issue up to now. If not, then perhaps extract these to a new PR?